.travis.yml

```diff
@@ -52,7 +52,7 @@ install:
 script:
   - vendor/bin/phpunit
   - if [[ $APP_ENV != 'postgres' ]]; then vendor/bin/behat --format=progress; fi
-  - if [[ $APP_ENV = 'postgres' ]]; then vendor/bin/behat --tags='~@sqlite||@postgres' --format=progress; fi
+  - if [[ $APP_ENV = 'postgres' ]]; then vendor/bin/behat --tags='~@sqlite' --format=progress; fi
```

Comment:
note that postgres tags are filtered by default in the `behat.yml.dist` now.

Reply:
You mean I need to restore `~@sqlite||@postgres`?

Reply:
I think it's good like this because CLI defined tags are replacing the ones from the config !